src/pages/cmDashboard.tsx

```diff
@@ -41,6 +42,10 @@ function cmDashboard() {
           </div>
         </div>
       </main>
+      <UploadImageComponent
```

Comment:
Please move this component to the `trials/[trials_id]/index.tsx` page.
Also make sure you've pulled from main, since it looks like this is the old cmDashboard.